compiler/ir/ir.psi2ir/src/org/jetbrains/kotlin/psi2ir/intermediate/PropertyLValue.kt

```diff
@@ -40,7 +40,7 @@ abstract class PropertyLValueBase(
     val superQualifier: IrClassSymbol?
 ) : LValue, AssignmentReceiver {
     override fun assign(withLValue: (LValue) -> IrExpression) =
-        callReceiver.call { dispatchReceiverValue, extensionReceiverValue ->
+        callReceiver.call { dispatchReceiverValue, extensionReceiverValue, _ ->
```

Comment:
Here we might loose context receiver information in compound assignment for a property (`a.b += c` where `b` is a property with context receiver).

Reply:
The function handles context receivers. This diff is outdated. [See the commit.](https://github.com/JetBrains/kotlin/commit/9d131e66f5028b7d02816eb9444468995d012a35)

Reply:
The corresponding test in [this commit](https://github.com/JetBrains/kotlin/pull/4488/commits/ead6d37aee92e05ba8b969241b1a63db034cc4b8).